tools/CMakeLists.txt

```diff
@@ -18,22 +18,34 @@ if(NOT Python3_FOUND)
 endif()
 
 function(add_groov_reg_lib target_name)
-    set(oneValueArgs INPUT_FILE OUTPUT_DIR)
-    set(multiValueArgs PERIPHERALS)
+    set(oneValueArgs INPUT_FILE OUTPUT_DIR BUS)
+    set(multiValueArgs PERIPHERALS INCLUDES)
     cmake_parse_arguments(ARG "" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
 
     set(python_script "${CMAKE_CURRENT_FUNCTION_LIST_DIR}/regs2groov.py")
 
     list(TRANSFORM ARG_PERIPHERALS APPEND ".hpp" OUTPUT_VARIABLE
                                                  register_headers)
 
+    if(DEFINED ARG_INCLUDES)
+        set(includes_arg "--includes;${ARG_INCLUDES}")
+    else()
+        set(includes_arg "")
+    endif()
+
+    if(DEFINED ARG_BUS)
+        set(bus_arg "--bus;${ARG_BUS}")
+    else()
+        set(bus_arg "")
+    endif()
```

Comment:
**suggestion**: Overwrite the existing variable, rather than creating a new one? Then there's no need for an `else()`, I think.